content-src/asrouter/templates/Trailhead/Trailhead.jsx

```diff
@@ -131,9 +132,33 @@ export class _Trailhead extends React.PureComponent {
     return str.value;
   }
 
+  onCardAction(action) {
+    let actionUpdates = {};
+
+    if (action.type === "OPEN_URL") {
+      let url = new URL(action.data.args);
+      let params = new URLSearchParams(url.search.slice(1));
+      params.append("utm_source", "activity-stream");
+      params.append("utm_campaign", "firstrun");
+      params.append("utm_medium", "referral");
+      params.append("utm_term", this.props.message.utm_term);
+
+      if (action.addFlowParams) {
+        params.append("flow_id", this.state.flowId);
```

Comment:
yeah, that probably needs updating. nice catch.

Reply:
I updated this but lets wait for @clouserw to confirm this change and whether we need to update the old flow too.

Reply:
Yes, the all the utm_parameters passed in the `metrics-flow` request should match those attached to the FxA URL. If by "old flow" you mean what will be the control, then yes its best to update that as well so that it includes the same parameters.

Reply:
Thanks @irrationalagent !